packages/jest-environment/src/index.ts

```diff
@@ -22,8 +22,17 @@ export type EnvironmentContext = Partial<{
   testPath: Config.Path;
 }>;
 
-// TODO: type this better: https://nodejs.org/api/modules.html#modules_the_module_wrapper
-type ModuleWrapper = (...args: Array<unknown>) => unknown;
+// Different Order than https://nodejs.org/api/modules.html#modules_the_module_wrapper , however needs to be in the form [jest-transform]ScriptTransformer accepts
+export type ModuleWrapper = (
+  module: Module,
+  exports: Module['exports'],
+  require: Module['require'],
+  __dirname: string,
+  __filename: Module['filename'],
+  global: Global.Global,
+  jest: Jest,
+  ...extraGlobals: Array<any>
```

Comment:
`unknown` over `any`?

Reply:
Actually the way its used in jest-runtime, it should be `Array<Global.Global[keyof Global.Global]>` which resolves to Array<any> anyways..

Reply:
that works as well 🙂